extensions/Worker.Extensions.Http.AspNetCore/src/WorkerBuilderExtensions.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Azure.Functions.Worker
     /// <summary>
     /// Provides extension methods to work with a <see cref="IFunctionsWorkerApplicationBuilder"/>.
     /// </summary>
-    public static class WorkerBuilderExtensions
+    internal static class WorkerBuilderExtensions
```

Comment:
Was chatting with @mattchenderson about the experience here. Should we flag those as obsolete with information about the new methods, potentially a link to the breaking change notes (perhaps even change implementation to throw with an exception that has all the details)?